microsoft-edge/webview2/concepts/print.md

```diff
@@ -0,0 +1,736 @@
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+* [CoreWebView2.ShowPrintUI() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui)
+* [CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.showprintui#microsoft-web-webview2-core-corewebview2-showprintui(microsoft-web-webview2-core-corewebview2printdialogkind))
+* [CoreWebView2.PrintAsync() Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printasync#microsoft-web-webview2-core-corewebview2-printasync)
+* [CoreWebView2.PrintToPdfAsync(resultFilePath, printSettings) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfasync#microsoft-web-webview2-core-corewebview2-printtopdfasync)
+* [CoreWebView2.PrintToPdfStreamAsync(printSettings) Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.printtopdfstreamasync#microsoft-web-webview2-core-corewebview2-printtopdfstreamasync)
+* [CoreWebView2PrintSettings Class](/dotnet/api/microsoft.web.webview2.core.corewebview2printsettings)
+* [CoreWebView2PrintDialogKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2printdialogkind)
+* [CoreWebView2PrintStatus Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2printstatus)
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+* [ICoreWebView2_16::ShowPrintUI(printDialogKind) method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#showprintui)
+* [ICoreWebView2_16::Print() method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#print)
+* [ICoreWebView2_7::PrintToPdf(resultFilePath, printSettings, handler) method](/microsoft-edge/webview2/reference/win32/icorewebview2_7#printtopdf)
+* [ICoreWebView2_16::PrintToPdfStream(printSettings, handler) method](/microsoft-edge/webview2/reference/win32/icorewebview2_16#printtopdfstream)
+* [ICoreWebView2PrintSettings Class](microsoft-edge/webview2/reference/win32/icorewebview2printsettings)
+* [ICoreWebView2PrintSettings2 Class](microsoft-edge/webview2/reference/win32/icorewebview2printsettings2)
+* [COREWEBVIEW2_PRINT_DIALOG_KIND Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_dialog_kind)
+* [COREWEBVIEW2_PRINT_STATUS Enum](/microsoft-edge/webview2/reference/win32/icorewebview2_16#corewebview2_print_status)
+* [ICoreWebView2PrintCompletedHandler](/microsoft-edge/webview2/reference/win32/icorewebview2printcompletedhandler)
+* [ICoreWebView2PrintToPdfCompletedHandler](/microsoft-edge/webview2/reference/win32/icorewebview2printtopdfcompletedhandler)
+* [ICoreWebView2PrintToPdfStreamCompletedHandler](/microsoft-edge/webview2/reference/win32/icorewebview2printtopdfstreamcompletedhandler)
+
+---
+
+<!-- end of tab-set -->
+
+
+<!-- ====================================================================== -->
+## See also
+
+* [Get started with WebView2](../get-started/get-started.md)
```

Comment:
```suggestion
* [Printing](./overview-features-apis.md#printing) in _Overview of WebView2 features and APIs_.
* [Get started with WebView2](../get-started/get-started.md)
```
If the present article is moved to /how-to/ dir instead of /concepts/ the link would be:
```
* [Printing](../concepts/overview-features-apis.md#printing) in _Overview of WebView2 features and APIs_.
```